main/main.ino

```diff
@@ -1037,106 +1049,105 @@ void loop() {
 
 #if defined(ESP8266) || defined(ESP32)
   if (WiFi.status() == WL_CONNECTED) {
+    ArduinoOTA.handle();
 #else
   if ((Ethernet.hardwareStatus() != EthernetW5100 && Ethernet.linkStatus() == LinkON) || (Ethernet.hardwareStatus() == EthernetW5100)) { //we are able to detect disconnection only on w5200 and w5500
 #endif
     failure_number_ntwk = 0;
-    if (client.connected()) {
+    if (!ProcessLock) { // If no blocking process we can handle MQTT processing
```

Comment:
What happens if this is set to true after this if check?

Reply:
in my point of view it is not possible until `ArduinoOTA.handle();` is called, and this call is done before this condition